.travis.yml

```diff
@@ -1,44 +1,30 @@
 services:
 - docker
 env:
   global:
   - REGISTRY_USER=ocadotechnologygitlab
-  - IMAGE_NAME=ocadotechnology/statsd
   - VCS_SOURCE="https://github.com/${TRAVIS_REPO_SLUG}"
-  - secure: YbNxrr1D1AkcqR6YGFvwQIkxD6X6fYt5qSr6Cc+yTVQgh81kaJEfz30iu39rfO3mb6kHvVnfxF2xOnF65gc67tShxvYQTMGwmRcmk5cDd+VKS23XPicvta/hrs90cIF6XQBAseoAqRGRyE8s70mELEb60GuwERvIkcJV/dppCYcIkf3H977OdtIwIAfWNTPoIVVhHID70Edjg33oJtoVu3NBeXZcJIJCw0vy649NBSrSgp2j+YZRxbaAUizSslVQJ1c6tYnP12dCzkHHVtC+mj5fWxwjTi//vwknckB4YFMeH7SVjU12VB6XYzjFYxrSLaWqCuwWjcDLrgx5RS0tAZQeD+sX5xCeXh8UenEsAvx5TyNLfuezlS3/+xZ6RKhtNOuWWT+/QgDHlpZ2s/LISqgchW2yWp0fbtnuz+WhLmSbtCTKVrR/++yDtk4QiiFJFG8WtI4903Cjg9A9INl3Zmiw9THuW8sHHJ6gGuaN8IPH9ydstWjx7Q62qDW+3nbEv6cpfF7MsVaD4IqYzj4JWv0ZU2ah9G5sk4WHFF4HtEHu3yO3EQuyShl8hYvPM/oG7bdGYASNBYkEBNEj/choaDD/1VV0jiCwegeWbxxVnSA8XsUiWsD5xs4KC6bh3lZPMqvzZpKxyh7NrAncjO0xjo2BCtzG8/E13uP6n/ZNXmg=
+  - SNYK_MODE=WARN
+  - secure: "YbNxrr1D1AkcqR6YGFvwQIkxD6X6fYt5qSr6Cc+yTVQgh81kaJEfz30iu39rfO3mb6kHvVnfxF2xOnF65gc67tShxvYQTMGwmRcmk5cDd+VKS23XPicvta/hrs90cIF6XQBAseoAqRGRyE8s70mELEb60GuwERvIkcJV/dppCYcIkf3H977OdtIwIAfWNTPoIVVhHID70Edjg33oJtoVu3NBeXZcJIJCw0vy649NBSrSgp2j+YZRxbaAUizSslVQJ1c6tYnP12dCzkHHVtC+mj5fWxwjTi//vwknckB4YFMeH7SVjU12VB6XYzjFYxrSLaWqCuwWjcDLrgx5RS0tAZQeD+sX5xCeXh8UenEsAvx5TyNLfuezlS3/+xZ6RKhtNOuWWT+/QgDHlpZ2s/LISqgchW2yWp0fbtnuz+WhLmSbtCTKVrR/++yDtk4QiiFJFG8WtI4903Cjg9A9INl3Zmiw9THuW8sHHJ6gGuaN8IPH9ydstWjx7Q62qDW+3nbEv6cpfF7MsVaD4IqYzj4JWv0ZU2ah9G5sk4WHFF4HtEHu3yO3EQuyShl8hYvPM/oG7bdGYASNBYkEBNEj/choaDD/1VV0jiCwegeWbxxVnSA8XsUiWsD5xs4KC6bh3lZPMqvzZpKxyh7NrAncjO0xjo2BCtzG8/E13uP6n/ZNXmg="
+  - secure: "oaLOoZzQTY/4t9wgEtKfX5m1v+tirvUFtc/gMrKUeBxoFaG07jnhr/X5p4LP1XgzK+eBknFU7O6/vsZw0UYZ8R5uszhi4ahP/ug7fSYvXT9dY2F9IAGspu8JkwoFPAFL0wikDOpNMiyMdCZKJNsAF3gyelnmomICLCKtuT2mjpSIkQTi+XyqViQ0o2nyPsWur82dR20es0YqQFpGlp+EM95XQEKDMFh8/7GcvoGtSo8JwEq/pWIlZCRwCm6NTSDUGujSNSiUsdvET1cJgoK+3Hzm7Ccex3JcmWFOjHAxHoEYR69LQm+7XNP7pdT1eYd/YlWqqUDXeqhDrerQj0Kv3ZwmIACFpbak1bJmVDg/7YCeQ3JZweCZpGS5I9NKHwbeDwwz3PhAkFAzGBfFjrgdZ91gmYaEYnuam8kBxBs5Y8VAoCft9nyqXbpQY7z9yB5kwydPPdUdYmUKabhHT4yarsvP3Efbnz2C6clmw//2BgmzfcL6XuCLLvG8d093v/TegtnVWm7AWo0/Gz2rqCSZSxbxdUC+ar1lSFLZatkd2DFi7PrDkqAe7OktvPz9a7E6KU1KhLX2x4qjxuw7XlpsM6n6Fz5u3w/xMT4ZeWkwiJWMIuuMhVMEDGW0httVEjvwFyYu3eFhTiSFWZHyHOe8kiVY15V3ZpfNtjsg+j1PtCc="
+  - secure: "meptwbsQls6iwpPaDT4z7wrgdeLt95iS/q/MY7L52YEua+7tknhCLOknZypZ9OzjpYb0CPQm6eRzG+g8TLNI3e0lKdR3skp7aumhDUPRhuXusYADrr14E7qBsfm3Kaahb3MaMKiKfJox0k2DBvEjlBq2zY+kHnJCj3BjzCte72ezqTu1G6yTHTXvuhmHwVkkpU5JjoT6UWF6l4Ot7AerBb970jAb/WK73l28vhhHzPTljLuYzdf+fI1N8f2DdPYhY0MeFeu85VNftWpmv4g7dsxlkOqPZDEWdJVEDO0zYQmpSlFGRZquRTN3YUgH8Rdezr5AFUOEE5ayXHRY7iVI1cT5e65mSkRYXhJMH47VdZhNtZ6oQ12OKP9cEEkd/ccwMh3Q0mAybuwElCxnucAMnYT5CT6+KNULWeVVTfOZmcJZbfw+3lsCqZ7AxGruEhMfrVDgH2Kk9BvF3yt0mABxqZR0u1+QqTiaaJcUi4akVa8C6kf4n5sflZMU5O+LLqE65trGi9YvId9Ir7opqZtDIkp/xdbi84/icVnO1AH9iGToGE+7mD3/kPbmqrnhRqjAC6WpwptSTmm7iYrw11De/tMx3A1rhH/+ohlaBiGCt3VR1rcBLcxr0rIR/El93wAqqn9VZgsDJ/BiWzQVJgt2QnU9frsa88K2i4vdp4MQQ0o="
```

Comment:
what is in these 3 secrets? not the actual values obviously

Reply:
I think the top one is the registry pass, and the other two are the snyk org and snyk token

Reply:
Hmmm, maybe the secrets could/should be commented so we know which is which?

Reply:
will do that now

Reply:
done

Reply:
thanks, sorry, would it be possible for the comments to match the environment vars they are?

Reply:
eg `#SNYK_ORG=` ?

Reply:
exactly

Reply:
That should be right now?